tests/unit/components/video-box/VideoBox.spec.jsx

```diff
@@ -1,19 +1,27 @@
 import { screen, render } from '@testing-library/react'
-
 import VideoBox from '~/components/video-box/VideoBox'
 
-const props = {
-  video: 'video.mp4'
-}
-
 describe('VideoBox component', () => {
-  beforeEach(() => {
-    render(<VideoBox { ...props } />)
+  const props = {
```

Comment:
all mocked data we extract out of the `describe` block and out of the `it` tests and write it above

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/7b4f3cde-a1e6-49c1-a488-22b2b947d84a)
fixed